devices/atlantic.js

```diff
@@ -0,0 +1,127 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const utils = require('../lib/utils');
+const assert = require('assert');
+const e = exposes.presets;
+const ea = exposes.access;
+
+
+const thermostatPositions = {
+    'barley_closed': 0,
+    'quarter_open': 1,
+    'half_open': 2,
+    'three_quarters_open': 3,
+    'fully_open': 4,
+    'swing': 5,
+};
+
+const energyMode = {
+    'eco': 0,
+    'normal': 1,
+    'powerfull': 2,
+};
+
+const tzLocal = {
+    quiet_fan: {
+        key: ['quiet_fan'],
+        convertSet: async (entity, key, value, meta) => {
+            assert(typeof value === 'boolean');
+            // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+            await entity.write('hvacFanCtrl', {0x1000: {value: 1, type: 0x10}});
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === 5) {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 1, type: 0x10}});
+            } else {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 0, type: 0x10}});
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4273: {value: index, type: 0x04}});
```

Comment:
There are 5 positions for ac louver position: open from 1 to 4, and swing.
I mad a mistake, in the code, I have 5 positions + swing. I will remove the `barley_closed` option.
For position 4:

From Hub:
```
Frame 2431: 54 bytes on wire (432 bits), 52 bytes captured (416 bits) on interface [\\.\pipe\zboss_sniffer_COM7](file:////pipe/zboss_sniffer_COM7), id 0
IEEE 802.15.4 Data, Dst: 0x0689, Src: 0x0000
ZigBee Network Layer Data, Dst: 0x0689, Src: 0x0000
ZigBee Application Support Layer Data, Dst Endpt: 1, Src Endpt: 1
    Frame Control Field: Data (0x40)
    Destination Endpoint: 1
    Cluster: Thermostat (0x0201)
    Profile: Home Automation (0x0104)
    Source Endpoint: 1
    Counter: 19
ZigBee Cluster Library Frame, Mfr: Atlantic Group (0x125b), Command: Write Attributes, Seq: 58
    Frame Control Field: Profile-wide (0x04)
    Manufacturer Code: Atlantic Group (0x125b)
    Sequence Number: 58
    Command: Write Attributes (0x02)
    Attribute Field, Enum8: 4
        Attribute: 0x4273
        Data Type: 8-Bit Enumeration (0x30)
        Uint8: 4 (0x04)
```

From zigbee2mqtt with my converter, I get the following error
```
Error: Write 0xredacted/1 hvacFanCtrl({"17011":{"value":4,"type":4}}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Write for 'BUFFER' not available)
```

Reply:
I temporarily commented out the line 47 to execute the line 49.

Reply:
Also add the manufacturerCode here and use the correct type: `type: 0x04` -> `type: 0x30`

Reply:
And the cluster is Thermostat (for a fan setting...)